### PR TITLE
Limit web access in Parent if flag set

### PR DESCRIPTION
**Pull request by armstrongnate**
refs: MBL-13884
affects: parent
release note: Disables links in web views if the setting is set in Canvas

Test plan:
* narmstrong.beta.instructure.com > s
* CS 1400 > Front Page > Tap 'Modules' link
* URL should load modally in a web view controller
* Try to tap a link to a module item or use course navigation to get around
* Links should not work

* twilson.beta.instructure.com > Observer1
* Assignment Overrides > Syllabus
* Tap 'Assignments' link
* URL should open in SFSafari
* Links should be tappable

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/cjuuynv) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fcjuuynv)
teacher: [Link](https://rebrand.ly/vyzsxku) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fvyzsxku)
parent: [Link](https://rebrand.ly/adnqyld) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fadnqyld)

File: Core/CoreTests/Views/CoreWebViewTests.swift [hunk 1]
```diff
@@ -192,4 +204,8 @@ class CoreWebViewTests: CoreTestCase {
         XCTAssertEqual(CoreWebView.htmlString("html"), "html")
         XCTAssertEqual(CoreWebView.htmlString("&'\"<>"), "&amp;&#39;&quot;&lt;&gt;")
     }
+
+    func testIsLinkNavigationEnabled() {
+
+    }
```
Comment (cobbal) [inline Core/CoreTests/Views/CoreWebViewTests.swift]: Is this supposed to be blank?

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: Core/CoreTests/Views/CoreWebViewTests.swift (hunk 1 above)
Comment (armstrongnate) [reply Core/CoreTests/Views/CoreWebViewTests.swift]: Nope, forgot I started down that path. Thanks.

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1021612&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1021613&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Comment by inst-danger** [comment]
#### Release Note: 
Disables links in web views if the setting is set in Canvas

#### Affected Apps: Parent

[MBL-13884](https://instructure.atlassian.net/browse/MBL-13884)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.61% | 93.61% | -0.01%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by bluwave** [comment]
IMG_ED39FD329509-1
I can pull this up with a force touch, is that ok?

**Comment by armstrongnate** [comment]
Hmm it's not ideal. I'll see if I can disable it. Good catch.

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1021809&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Pull request merged**